hr_utilization_analysis/report/hr_utilization_analysis.py

```diff
@@ -108,82 +178,20 @@ class HrUtilizationAnalysisEntry(models.TransientModel):
         related="employee_id.parent_id",
         store=True,
     )
-    date = fields.Date(string="Date", required=True)
+    project_id = fields.Many2one(string="Project", comodel_name="project.project")
```

Comment:
Please give more details on adding Project, since we were just using grouping lines by project in order to get same thing?

Reply:
Hm, if you group on line_ids.project_id, it filters but you dont get differentiation in values between one project and another?

Now we can see how many hours written on one project and on another, even if they are for same person on same date:

![image](https://user-images.githubusercontent.com/1466356/100896163-a1ec9880-34be-11eb-845b-46dd058f21da.png)

Reply:
Ah, true. Makes sense 👍